codingTest_kew/12.11_완주하지_못한_선수.js

```diff
@@ -0,0 +1,11 @@
+function solution(participant, completion) {
+participant.sort() 
+completion.sort()
+let obj1 = {...participant}
+let obj2 = {...completion}
+for(let num1 = 0; num1 <= 100000; num1++){
```

Comment:
```suggestion
for(let num1 = 0; num1 <= 100000; num1++){  //이 부분
```
문제가 정확히 기억은 안나는데 어짜피 완주한 사람은 참여자보다 적을테니 for문의 조건문이 num1이 참여자의 길이보다 작거나 같을 때로 바꿔도 되지 않을까요??